woocommerce_fusion/tasks/sync_sales_orders.py

```diff
@@ -610,7 +614,8 @@ def create_missing_items(self, items_list, woocommerce_site):
 			if not found_item:
 				# Create Item
 				item = frappe.new_doc("Item")
-				item.item_code = _("{0}").format(item_woo_com_id)
+				# item.item_code = _("{0}").format(item_woo_com_id)
+				item.item_code = item_woo_com_id
 				item.stock_uom = self.settings.uom or _("Nos")
 				item.item_group = self.settings.item_group
 				item.item_name = item_data.get("name")
```

Comment:
Line 623:
`row.woocommerce_id = item_woo_com_id`
is going to be problematic if left as such. `woocommerce_id` represents the Woocommerce item `product_id`. Our code has changed `item_woo_com_id` variable from the _either_ `product id` or `sku`.

I guess you need two variables then; one for the product id and another for the item name (which will either be the sku or product id)